packages/haiku-serialization/src/bll/Project.js

```
@@ -320,7 +320,7 @@ class Project extends BaseModel {
   emitHook (...args) {
     const method = args.shift()
     const metadata = args.pop()
-    if (metadata.from === this.getAlias()) {
+    if (metadata && metadata.from === this.getAlias()) {
```

Spooky that sometimes `metadata` isn't present?

---

Yes…but looks like this isn't due to a change in this PR, correct? I filed a [ticket](https://app.asana.com/0/480796620059175/514892342431432/f) because this is probably a bug.

---

No, it's not due to this PR. The issue is also noted on this other [ticket by Zack](https://app.asana.com/0/480796620059175/506363302552360). So my "fix" here takes care of the symptom, but perhaps there's something ickier happening when `metadata` isn't present. Let me know if you'd like me to leave this check-in, or let the error ride until you look into that other ticket you filed.

---

Yeah, I think in this case since we're already tracking it, it's good to suppress the error and helps us get a release out sooner. I haven't noticed any awful side effects from this with some light testing.